make/TestImage.gmk

```diff
@@ -31,9 +31,9 @@ include MakeBase.gmk
 ############################################################################
 
 ifeq ($(call isTargetOs, windows), true)
-  FIXPATH_COPY := $(TEST_IMAGE_DIR)/bin/fixpath.exe
+  FIXPATH_COPY := $(TEST_IMAGE_DIR)/bin/fixpath.sh
```

Comment:
I assume you are right. Even though it bothers me to my bone that we just check out the complete source for testing, instead of building a proper test image. :) I'll fix.